package.json

```diff
@@ -11,13 +11,13 @@
   },
   "scripts": {
     "lint": "yarn eslint . && yarn prettier --list-different .",
-    "mass-merge": "mass-merge"
+    "mass-merge": "node index.js"
```

Comment:
Is this intentional?

Reply:
Yup - a last minute change 😅 iirc that line was added to be able to run the command in the mass-merge directory, to try out changes, but it didn't seem to work (with yarn)
With this it's possible to run `yarn mass-merge [args]` in this repo's dir